src/api/gravidkrav/mapGravidKravRequest.test.ts

```diff
@@ -0,0 +1,151 @@
+import { mapGravidKravRequest } from './mapGravidKravRequest';
+import { parseDato } from '../../utils/Dato';
+import { Arbeidsgiverperiode, GravidKravRequest } from './GravidKravRequest';
+
+describe('mapGravidKravRequest', () => {
+  it('should throw error when missing fnr', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        undefined,
+        '456',
+        parseDato('01.02.2020'),
+        parseDato('03.04.2025'),
+        5,
+        3000,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
+  it('should throw error when missing orgnr', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        '123',
+        undefined,
+        parseDato('01.02.2020'),
+        parseDato('03.04.2025'),
+        5,
+        3000,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
+  it('should throw error when error in fra', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        '123',
+        '123',
+        parseDato('99.02.2020'),
+        parseDato('03.04.2025'),
+        5,
+        3000,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
+  it('should throw error when error in til', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        '123',
+        '123',
+        parseDato('01.02.2020'),
+        parseDato('99.04.2025'),
+        5,
+        3000,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
+  it('should throw error when empty dager', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        '123',
+        '123',
+        parseDato('01.02.2020'),
+        parseDato('02.04.2025'),
+        undefined,
+        3000,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
+  it('should throw error when empty beløp', () => {
+    expect(() => {
+      mapGravidKravRequest(
+        '123',
+        '123',
+        parseDato('01.02.2020'),
+        parseDato('02.04.2025'),
+        2,
+        undefined,
+        'dokumentasjon',
+        true
+      );
+    }).toThrowError();
+  });
+
```

Comment:
Kunne kanskje tilføye en test på 'should not throw error when empty dokumentasjon' her?

Reply:
Godt poeng. Jeg fikser.